config.json

```diff
@@ -19,9 +19,15 @@
     "average_run_time": 3.0
   },
   "files": {
-    "solution": ["%{kebab_slug}.v"],
-    "test": ["run_test.v"],
-    "example": [".meta/example.v"],
+    "solution": [
+      "%{kebab_slug}.v"
```

Comment:
Quick question: is there any particular reason for the solution file being in kebab case while the test file is in snake case?

I couldn't tell if this is a V convention because they seem to use snake case across the V source code.

Reply:
the solution file is kebab case to match with [exercism norms](https://github.com/exercism/problem-specifications/tree/main/exercises). however, v [needs snake case for testing](https://github.com/exercism/vlang#name-of-testing-script-for-each-solution), hence why testing is snake case.